<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpperMachineSetting</class>
 <widget class="QWidget" name="UpperMachineSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Setup_UpperMachineSetting_lbBg">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/wifibg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Setup_UpperMachineSetting_lbMachine">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(215, 215, 215);</string>
   </property>
   <property name="text">
    <string>仪器列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Setup_UpperMachineSetting_BtnModify">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/updatebt.png);</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Setup_UpperMachineSetting_BtnFlush">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>590</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/userback.png);</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Setup_UpperMachineSetting_BtnBack">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>720</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/userback.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Setup_UpperMachineSetting_leNo">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Setup_UpperMachineSetting_lbNo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Setup_UpperMachineSetting_leName">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Setup_UpperMachineSetting_lbName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>名称:</string>
   </property>
  </widget>
  <widget class="QTableView" name="Setup_UpperMachineSetting_tbMachine">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>256</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(211, 0, 0);
selection-color: rgb(204, 0, 0);
background-color: rgb(239, 239, 239);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
